src/matlab2tikz.m

```diff
@@ -435,11 +435,11 @@ function matlab2tikz(varargin)
 
     % Alternative Positioning of axes.
     % Select relevant Axes and draw them.
-    [relevantAxesHandles, axesBoundingBox] = getRelevantAxes(m2t, axesHandles);
+    [m2t, axesBoundingBox] = getRelevantAxes(m2t, axesHandles);
```

Comment:
That's a valid point! Please add a short comment that documents that proposed purpose to the code.